### PR TITLE
Add requests-negotiate-sspi

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rzillman** [comment]
I am willing to be a maintainer.

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
@brandond if you would like to also be added as a maintainer of the conda-forge recipe just post here to let us know and it can be done either here or in a subsequent PR.

Being a maintainer of the recipe means you will get GitHub notifications for the PRs which are automatically opened when new versions are published to pypi and it will give you to ability to merge those PRs.

*Note:* Whilst the PRs are opened automatically, the deps aren't updated so part of being a maintainer currently involves manually updating any changed deps.

File: recipes/requests-negotiate-sspi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brandond/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1566084baec815d11f2df07f01e078343a4bbbbb6e5ba163cc9e1b5f7d751f0f
+
+build:
+  skip: True  # [not win]
```
Comment (chrisburr) [inline recipes/requests-negotiate-sspi/meta.yaml]: Can this be noarch? It’s okay if it’s missing dependencies on windows

Comment (dhirschfeld) [reply recipes/requests-negotiate-sspi/meta.yaml]: IIUC it's a Windows only package. As it is, it's implemented via `pywin32` so is currently Windows only.

**Pull request merged**